build/linux/dist/install.sh

```diff
@@ -141,10 +144,18 @@ simple_uninstall_f() {
     rm "${HOME}/.local/share/applications/arduino.desktop"
   fi
 
+  if [ -f "${HOME}/.local/share/applications/arduino-arduinoide.desktop" ]; then
+    rm "${HOME}/.local/share/applications/arduino-arduinoide.desktop"
+  fi
+
```

Comment:
This should have a comment stating this was the name used in older scripts (perhaps add an OLD_RESOURCE_NAME var?)